src/board.py

```diff
@@ -0,0 +1,18 @@
+def create_board():
+    return [[0 for j in range(5)] for i in range(5)]
+
+
+def get_symbol(value):
+    if value < 0:
+        return 'x'
+    if value > 0:
+        return 'o'
+    return ' '
+
+
+def print_board(board):
+    for line in board:
+        for tile in line:
+            symbol = get_symbol(tile)
+            print('[' + symbol + ']', end='')
```

Comment:
Protip : instead of concatenating strings, you can use literal strings:
https://www.python.org/dev/peps/pep-0498/

```python
print(f'[{symbol}]', end='')
```

Or like the old school %:

```python
print('[%s]' % symbol, end='')
```
It is far more readable.

Reply:
Done on next PR (wip): https://github.com/marmelab/quixo-python/pull/3